src/Tabs/Tab.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { ButtonNoStyle } from '../Button/Button';
+import Span from '../Span';
+
+const StyledBtn = ButtonNoStyle.extend`
+  cursor: pointer;
+  background: #EEF1F3;
+  border-right: solid 1px #CFCFCF;
+  border-bottom: solid 1px #CFCFCF;
+  &:first-child {
+    ${props => props.rounded && 'border-radius: 6px 0 0 0;'}
+  }
+  &:last-child {
+    border-right: none;
+    ${props => props.rounded && 'border-radius: 0 6px 0 0;'}
+  }
+  ${props => props.active && `
+    border-bottom: 1px solid transparent;
+    background: #FFFFFF;
+    `
+  }
+  border-top: none;
+  flex: 1;
+  > ${Span} {
+    > * {
+      display: flex;
+      align-items: center;
+      svg {
+        padding-right: 1rem;
+        fill: currentColor;
+      }
+    }
+    color: #017AC9;
+    font-size: 1.1rem;
+    text-transform: uppercase;
+    margin: 0 2rem 1rem 2rem;
+    padding-top: 0.75rem;
+    display: flex;
+    justify-content: center;
+    border-top: 4px solid transparent;
+    ${props => props.active && `
+      border-top: solid 4px #017AC9;
+      color: #000000;
+    `}
+    ${props => props.disabled && 'color: #B7B7B7;'}
+  }
+`;
+
+StyledBtn.propTypes = {
+  className: PropTypes.string,
+  active: PropTypes.bool,
+  disabled: PropTypes.bool,
+  rounded: PropTypes.bool
+};
+
+const Tabs = ({
+  header,
+  children, // eslint-disable-line no-unused-vars
```

Comment:
Removed this line, but kept the prop validation.
Children is not used in Tab component.
Tabs component uses Tab component's children.